core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -202,7 +203,7 @@ class H2OContext private(val sparkSession: SparkSession, conf: H2OConf) extends
   /** Transforms Dataset[Supported type] to H2OFrame */
   def asH2OFrame[T<: Product : TypeTag](ds: Dataset[T]): H2OFrame = asH2OFrame(ds, None)
   def asH2OFrame[T<: Product : TypeTag](ds: Dataset[T], frameName: Option[String]): H2OFrame =
-    DatasetConverter.toH2OFrame(this, ds, frameName)
+    withConversionDebugPrints(sparkContext, "Dataset", DatasetConverter.toH2OFrame(this, ds, frameName))
```

Comment:
Curious question, could we use Spark to collect these stats? They have metric system in place already.

Reply:
We could probably measure time of the conversion job and also the rest of related tasks. In Spark we can measure jobs, tasks at better granularity then we do now.

This measures the conversion and also all the related tasks in a quick easy solutionI can play with it and try to re-use spark for it, it would be nicer solution for sure.

This was just to have something quickly for the release :)